invenio_app_ils/search/api.py

```diff
@@ -21,6 +21,10 @@ class Meta:
         index = "documents"
         doc_types = None
 
+    def search_by_pid(self, *pids):
```

Comment:
could it be done with https://github.com/inveniosoftware/invenio-search/blob/e817a752d9e98680486dddd022e4d7d392f5ee6b/invenio_search/api.py#L113 ?